### DISCUSSION TITLE
[Projects Beta] Allow views to have different issues

### DISCUSSION THREAD

**Discussion by nrod** [community/community#9623] [Projects and Issues] [answered]
Would it be possible (or is it already and I could not find it) to have different sets of issues in different views of the same project?
Moving issues between views should be also possible. Copying I already checked and works fine.
At the moment I need to create several projects to behave like the views below.

> Example:
Project: **AllMyWork**
> 
> Views: 
**IssuesForJan**: 12 issues
**IssuesForFeb**: 4 issues
...
**Backlog**: 84 issues

**Comment by ghost** [comment, marked as answer]
If I understand you correctly this is already possible with view filters.

For your example:
1. create a field called month
2. create 2 views, filter 1 by "month: jan", the other  by "month:feb"

**Reply by nrod** [reply]
Thank you @darian-nftychat! I already set it up in the meantime and it does work as you say.
